smart-contracts/contracts-common/concordium-contracts-common/src/schema.rs

```diff
@@ -1057,6 +1057,8 @@ mod impls {
         NoReturnValueInReceive,
         #[error("Return values not supported for this module version")]
         ReturnValueNotSupported,
+        #[error("Event schema not found in contract schema")]
+        NoErrorInEvents,
```

Comment:
The name is misleading, consider:
```suggestion
        NoEventInContract,
```

Reply:
Regarding both this and the comment below I initially called it `NoEventInContract` (see former commit). But an earlier comment ask to replace this with those which was already there and follow the naming pattern og the other errors - hence `NoError…`.